src/common/file_util.cpp

```diff
@@ -112,6 +122,14 @@ bool Exists(const std::string& filename) {
 }
 
 bool IsDirectory(const std::string& filename) {
+#ifdef ANDROID
+    if (!AndroidStorage::IsDirectory(filename)) {
+        LOG_DEBUG(Common_Filesystem, "stat failed on {}", filename);
+        return false;
+    }
+    return true;
+#endif
+
```

Comment:
```suggestion
#ifdef ANDROID
    return AndroidStorage::IsDirectory(filename);
#endif
```
the Log message does not make sense to me, it's fine if something is not a directory.